Makefile

```diff
@@ -390,7 +390,7 @@ fsm_graph: clean
 .PHONY: clean
 clean:
 	@echo "cleaning up caches and output"
-	go clean -cache -testcache -r -x ./... 2>&1 >/dev/null
+	go clean -cache -testcache -r
```

Comment:
I'm with Peter on this one (referencing his comments from the core version of this patch). I think dev/null handling should stay.

Reply:
Upon further investigation, we don't need the redirect as -x is what was generating the noise. +1, will merge shortly.